doc/model/train-se-e2-r.md

```diff
@@ -2,6 +2,56 @@
 
 The notation of `se_e2_r` is short for the Deep Potential Smooth Edition (DeepPot-SE) constructed from the radial information of atomic configurations. The `e2` stands for the embedding with two-atom information.
 
+## Theory
+
+The descriptor, using either radial-only information, is given by
+
+```math
+    \mathcal{D}^i = \frac{1}{N_c} \sum_j (\mathcal{G}^i)_{jk},
+```
+
+where
+$N_c$ is the expected maximum number of neighboring atoms, which is the same constant for all atoms over all frames.
+A matrix with a dimension of $N_c$ will be padded if the number of neighboring atoms is less than $N_c$.$\mathcal{R}^i \in \mathbb{R}^{N_c}$ is the coordinate matrix, and each row of $\mathcal{R}^i$ can be constructed as
```

Comment:
... is less than $N_c$.$\mathcal{R}^i \in ...
There should be a space between the period and the $\mathcal{R}.

Reply:
```suggestion
A matrix with a dimension of $N_c$ will be padded if the number of neighboring atoms is less than $N_c$. $\mathcal{R}^i \in \mathbb{R}^{N_c}$ is the coordinate matrix, and each row of $\mathcal{R}^i$ can be constructed as
```